ast/term.go

```diff
@@ -1920,7 +1936,7 @@ func (obj *object) Hash() int {
 
 // IsGround returns true if all of the Object key/value pairs are ground.
 func (obj *object) IsGround() bool {
-	return obj.ground == 2*len(obj.keys)
+	return obj.ground == 2*len(obj.sortedKeys())
```

Comment:
Do we care if they are sorted or not? Just wondering if there's a shortcut here.